### ISSUE TITLE
Installing iRODS 4.1.10 with Oracle database plugin as RPM has dependency errors with Oracle Client installed from Oracle package

### ISSUE THREAD

**Issue by kript** [irods/irods#3863]
### iRODS Version, OS and Version

iRODS 4.1.10
Red Hat 7.4
Oracle Instant Client  11.1-basic

### What did you try to do?

Have self hosted a iRODS 4.x Repository in our internal satellite setup, with Oracle instant client pre-installed, RPM direct from Oracle.

` yum install irods-icat irods-dev irods-database-plugin-oracle`

### Expected behavior

iRODS is installed

### Observed behavior

```
# echo $ORACLE_HOME
/usr/lib/oracle/11.1/client64
# echo $LD_LIBRARY_PATH
/usr/lib/oracle/11.1/client64
rpm -qa | grep oracle
oracle-instantclient11.1-basic-11.1.0.7.0-1.x86_64
 yum install irods-icat irods-dev irods-database-plugin-oracle
Loaded plugins: package_upload, product-id, search-disabled-repos,
              : subscription-manager
...
Resolving Dependencies
--> Running transaction check
---> Package irods-database-plugin-oracle.x86_64 0:1.10-0 will be installed
--> Processing Dependency: libocci.so.11.1()(64bit) for package: irods-database-plugin-oracle-1.10-0.x86_64
--> Processing Dependency: libnnz11.so()(64bit) for package: irods-database-plugin-oracle-1.10-0.x86_64
--> Processing Dependency: libclntsh.so.11.1()(64bit) for package: irods-database-plugin-oracle-1.10-0.x86_64
---> Package irods-dev.x86_64 0:4.1.10-0 will be installed
---> Package irods-icat.x86_64 0:4.1.10-0 will be installed
--> Finished Dependency Resolution
Error: Package: irods-database-plugin-oracle-1.10-0.x86_64 (SangerHPC_IRODS_IRODS_RHEL7)
           Requires: libnnz11.so()(64bit)
Error: Package: irods-database-plugin-oracle-1.10-0.x86_64 (SangerHPC_IRODS_IRODS_RHEL7)
           Requires: libclntsh.so.11.1()(64bit)
Error: Package: irods-database-plugin-oracle-1.10-0.x86_64 (SangerHPC_IRODS_IRODS_RHEL7)
           Requires: libocci.so.11.1()(64bit)
**********************************************************************
yum can be configured to try to resolve such errors by temporarily enabling
disabled repos and searching for missing dependencies.
To enable this functionality please set 'notify_only=0' in /etc/yum/pluginconf.d/search-disabled-repos.conf
**********************************************************************

Error: Package: irods-database-plugin-oracle-1.10-0.x86_64 (SangerHPC_IRODS_IRODS_RHEL7)
           Requires: libnnz11.so()(64bit)
Error: Package: irods-database-plugin-oracle-1.10-0.x86_64 (SangerHPC_IRODS_IRODS_RHEL7)
           Requires: libclntsh.so.11.1()(64bit)
Error: Package: irods-database-plugin-oracle-1.10-0.x86_64 (SangerHPC_IRODS_IRODS_RHEL7)
           Requires: libocci.so.11.1()(64bit)
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
[root@irods-seq-ies1 ~]# yum install irods-icat irods-dev irods-database-plugin-oracle --skip-broken
Loaded plugins: package_upload, product-id, search-disabled-repos,
              : subscription-manager
Resolving Dependencies
--> Running transaction check
---> Package irods-database-plugin-oracle.x86_64 0:1.10-0 will be installed
--> Processing Dependency: libocci.so.11.1()(64bit) for package: irods-database-plugin-oracle-1.10-0.x86_64
--> Processing Dependency: libnnz11.so()(64bit) for package: irods-database-plugin-oracle-1.10-0.x86_64
--> Processing Dependency: libclntsh.so.11.1()(64bit) for package: irods-database-plugin-oracle-1.10-0.x86_64
---> Package irods-dev.x86_64 0:4.1.10-0 will be installed
---> Package irods-icat.x86_64 0:4.1.10-0 will be installed
SangerHPC_IRODS_IRODS_RHEL7/filelists                  | 113 kB     00:00     
--> Running transaction check
---> Package irods-dev.x86_64 0:4.1.10-0 will be installed
--> Running transaction check
---> Package irods-dev.x86_64 0:4.1.10-0 will be installed
--> Finished Dependency Resolution

Packages skipped because of dependency problems:
    irods-database-plugin-oracle-1.10-0.x86_64 from SangerHPC_IRODS_IRODS_RHEL7
    irods-icat-4.1.10-0.x86_64 from SangerHPC_IRODS_IRODS_RHEL7
Error: Success - deps resolved
**********************************************************************
yum can be configured to try to resolve such errors by temporarily enabling
disabled repos and searching for missing dependencies.
To enable this functionality please set 'notify_only=0' in /etc/yum/pluginconf.d/search-disabled-repos.conf
**********************************************************************


Dependencies Resolved

==============================================================================
 Package                    Arch   Version  Repository                   Size
==============================================================================
Installing:
 irods-dev                  x86_64 4.1.10-0 SangerHPC_IRODS_IRODS_RHEL7  38 M
Skipped (dependency problems):
 irods-database-plugin-oracle
                            x86_64 1.10-0   SangerHPC_IRODS_IRODS_RHEL7 3.8 M
 irods-icat                 x86_64 4.1.10-0 SangerHPC_IRODS_IRODS_RHEL7  17 M

Transaction Summary
==============================================================================
Install                        1 Package
Skipped (dependency problems)  2 Packages

Total download size: 38 M
Installed size: 283 M
Is this ok [y/d/N]: n
Exiting on user command
```

### Steps to reproduce

Attempt to install Oracle database client for iRODS as RPM, with Oracle Instant Client already installed as RPM.

### Steps to Workaround

Download the RPM's manually and run with `--nodeps`
`rpm -i --nodeps  irods-icat-4.1.10-centos7-x86_64.rpm  irods-database-plugin-oracle-1.10-centos7.rpm`

**Comment by kript** [comment]
Its worth noting that the Oracle database plugin on Centos has an unmarked dependency on `sqlplus` being installed, but I'm not sure if this is the right place to note that, or if a new issue is the right thing, since Oracle package that separately..

**Comment by trel** [comment]
Isn't this a matter of the oracle instant client packages being broken?

I think the oracle client package declares the `sqlplus` that dependency, no?

What is the action for us to take here?

**Comment by kript** [comment]
I think its  the oracle database plugin package requiring a dependency on the sqlplus package.

**Comment by alanking** [comment]
See #8830
